oak/server/asylo/BUILD

```diff
@@ -142,3 +143,22 @@ enclave_loader(
         "@com_google_asylo//asylo/util:logging",
     ],
 )
+
+container_image(
+    name = "oak-docker",
+    base = "@cc_image//image",
+    entrypoint = [
+        "./oak_host_loader",
+        "--enclave_path=oak_enclave_debug.so",
+        "--grpc_port=8888",
+    ],
+    files = [
+        "oak_enclave_debug.so",
+        "oak_host_loader",
+    ],
+    ports = [
+        "8080/tcp",  # Application port.
```

Comment:
It would be good to add a comment pointing to where these ports are defined in the code currently.

Reply:
Done